src/core/dbus-manager.c

```diff
@@ -2092,11 +2092,18 @@ static int method_enable_unit_files_generic(
         if (r < 0)
                 return r;
 
-        r = sd_bus_message_read(message, "bb", &runtime, &force);
-        if (r < 0)
-                return r;
+        if (sd_bus_message_is_method_call(message, NULL, "EnableUnitFilesWithFlags")) {
+                r = sd_bus_message_read(message, "t", &flags);
```

Comment:
"t" requires a uint64_t. i.e. you probably should decode into some temp var first, and then after validating the mask convert it to the `UnitFileFlags` type

Reply:
fixed